build.gradle

```diff
@@ -136,9 +136,6 @@ subprojects {
         tasks.withType(JavaCompile).all {
             options.compilerArgs << "-Werror"
         }
-        tasks.withType(KotlinCompile).all {
```

Comment:
Was this accidentally removed?

Reply:
No, since deprecation result in warnings we can no longer fail the build